src/ts/playerutils.ts

```diff
@@ -60,26 +60,25 @@ export namespace PlayerUtils {
   }
 
   /**
-   * Returns seekable time range.
-   * For live streams it will calculate timestamp timerange if tweak
-   * `enable_seek_for_live` is not present
+   * Calculates player seekable time range for live.
+   * For live streams we need to calculate range since we receive { start: -1, end: -1 }
+   * unless `enable_seek_for_live` tweak is present
+   *
+   * @param player
    */
-  export function getSeekableRange(player: PlayerAPI): TimeRange {
-    let {start, end} = player.getSeekableRange();
-
-    // If it is VOD or we have `enable_seek_for_vod` enabled
-    if (!player.isLive() || end > -1 && start > -1) {
-      return {start, end};
+  export function getSeekableRangeForLive(player: PlayerAPI): TimeRange {
+    if (!player.isLive()) {
+      return player.getSeekableRange();
```

Comment:
will still return `undefined` on mobile SDKs. You need to check if this is implemented and I don't know return an empty object or `0` values 😕 Whatever works for the seekbar.

Reply:
Well, it is basically supposed to work only for live, that is some fallback there. In this implementation, we are using it on markers only if we are live as well, so we should not be facing any problems here, as it will always go through calculation.

Reply:
👍